wiki/Gameplay/Game_modifier/Auto/de.md

```diff
@@ -0,0 +1,62 @@
+---
+stub: true
+tags:
+  - auto
+  - autoplay
+  - auto play
+  - AT
+  - AO
+  - mod
+  - game modifier
+  - Spielmodifikation
+---
+
+# Auto (Mod)
+
+![AT Modsymbol](/wiki/shared/mods/AT.png "Auto (AT) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*\
+*Nicht zu verwechseln mit [Autopilot (Mod)](/wiki/Game_modifier/Autopilot).*
+
+## Übersicht
+
+- Abkürzung: AT oder AO
+- Typ: Sonstige
+- Score-Multiplikator: 1,00 x
```

Comment:
Now that I think about it, in my opinion, it's better to not put a space between the number and the x. The article `Co-op mod` has a table with numerous multipliers and I think putting a space everywhere makes the table less readable.

On the contrary, where I do think it to make sense is in the description of dimensions with the so called `Malkreuz` (e.g. `1920 × 1080`).

Very nitpicky, I know, but I wanted to mention it.

Reply:
```suggestion
- Score-Multiplikator: 1,00x
```